router/packetconn.go

```diff
@@ -48,6 +48,7 @@ func (r *Router) newLocalPeer() *peer {
 // was delivered using tree routing).
 func (r *Router) ReadFrom(p []byte) (n int, addr net.Addr, err error) {
 	var frame *types.Frame
+	n = 0
```

Comment:
Is this necessary? `n` should already be initialised to an empty value for `int`, i.e. zero.